### DISCUSSION TITLE
add custom fields to frontmatter

### DISCUSSION THREAD

**Discussion by nosovk** [requarks/wiki#5453] [General]
In our MD we use additional frontmatter filelds.
like:
```
description: Some text
yearOfManufacture: 2014
typeOfCertification: EU only
```
Unfortunately when we try to edit our MD's with Wiki.JS there is no way to see those additional frontmatter fields. Also, when we update page in wikijs - those fields removed from original MD.

It would be nice to have a proper way of working with frontmatter.

**Comment by celsobessa** [comment]
Same use case for me. I would appreciate this feature as well.
